### ISSUE TITLE
[BUG] PDF-Preview in Edit View shows only unicode boxes instead of normal characters

### ISSUE THREAD

**Issue by lagset** [paperless-ngx/paperless-ngx#3890]
### Description

I just uploaded a pretty standard terms and conditions PDF (digital, with inline text) to paperless (via consume folder). This is a first time this occured in over 600 documents added to my paperless instance.

However, in Edit View the document is displayed like this:
![image](https://github.com/paperless-ngx/paperless-ngx/assets/15156011/5ccec6c6-df07-48dd-8670-59222ca76bdf)

I then thought this may be connected to OCR and thus not a paperless issue. But, it does not make any difference downloading the archived vs the original document: Both show the characters correctly.

![image](https://github.com/paperless-ngx/paperless-ngx/assets/15156011/a051728a-405f-4fab-8972-9fd6ce2001a9)
Screenshot of the archived PDF: All characters (german document) display as they should.

And even in any other UI Element in paperless, the document is displayed as expected, e.g. in the "hover over show icon"-preview in the document grid:
![image](https://github.com/paperless-ngx/paperless-ngx/assets/15156011/12b23ec5-150c-481a-97da-412bf91a63a9)

P.S. If I am wrong and this isn't an issue concerning paperless-ngx feel free to close this bug. I just thought the inconsistent display behavior may be relevant.

### Steps to reproduce

1. Upload this document to paperless [Allgemeine Geschäftsbedingungen.pdf](https://github.com/paperless-ngx/paperless-ngx/files/12207131/Allgemeine.Geschaftsbedingungen.pdf)
2. Edit the document in paperless and check the preview section

### Webserver logs

```bash
[2023-07-29 12:02:58,701] [INFO] [paperless.management.consumer] Adding /usr/src/paperless/consume/Allgemeine Geschäftsbedingungen.pdf to the task queue.

[2023-07-29 12:02:59,438] [INFO] [paperless.consumer] Consuming Allgemeine Geschäftsbedingungen.pdf

[2023-07-29 12:02:59,442] [DEBUG] [paperless.consumer] Detected mime type: application/pdf

[2023-07-29 12:02:59,443] [DEBUG] [paperless.consumer] Parser: RasterisedDocumentParser

[2023-07-29 12:02:59,448] [DEBUG] [paperless.consumer] Parsing Allgemeine Geschäftsbedingungen.pdf...

[2023-07-29 12:02:59,672] [DEBUG] [paperless.parsing.tesseract] Calling OCRmyPDF with args: {'input_file': PosixPath('/tmp/paperless/paperless-ngxo_esd8o_/Allgemeine Geschäftsbedingungen.pdf'), 'output_file': PosixPath('/tmp/paperless/paperless-3dnd_f_7/archive.pdf'), 'use_threads': True, 'jobs': 2, 'language': 'deu', 'output_type': 'pdfa', 'progress_bar': False, 'skip_text': True, 'clean': True, 'deskew': True, 'rotate_pages': True, 'rotate_pages_threshold': 12.0, 'sidecar': PosixPath('/tmp/paperless/paperless-3dnd_f_7/sidecar.txt')}

[2023-07-29 12:03:00,482] [DEBUG] [paperless.parsing.tesseract] Incomplete sidecar file: discarding.

[2023-07-29 12:03:00,799] [DEBUG] [paperless.consumer] Generating thumbnail for Allgemeine Geschäftsbedingungen.pdf...

[2023-07-29 12:03:00,806] [DEBUG] [paperless.parsing] Execute: convert -density 300 -scale 500x5000> -alpha remove -strip -auto-orient /tmp/paperless/paperless-3dnd_f_7/archive.pdf[0] /tmp/paperless/paperless-3dnd_f_7/convert.webp

[2023-07-29 12:03:04,419] [DEBUG] [paperless.consumer] Saving record to database

[2023-07-29 12:03:04,419] [DEBUG] [paperless.consumer] Creation date from parse_date: 2021-10-01 00:00:00+00:00

[2023-07-29 12:03:04,973] [DEBUG] [paperless.matching] Correspondent ******************* matched on document 2021-10-01 Allgemeine Geschäftsbedingungen because it contains this string: "*******************"

[2023-07-29 12:03:04,986] [INFO] [paperless.handlers] Assigning correspondent ******************* to 2021-10-01 Allgemeine Geschäftsbedingungen

[2023-07-29 12:03:05,093] [DEBUG] [paperless.matching] DocumentType ********* matched on document 2021-10-01 ******************* Allgemeine Geschäftsbedingungen because it contains this string: "*********"

[2023-07-29 12:03:05,094] [DEBUG] [paperless.matching] DocumentType ********* matched on document 2021-10-01 ******************* Allgemeine Geschäftsbedingungen because it contains this string: "*********"

[2023-07-29 12:03:05,148] [DEBUG] [paperless.matching] DocumentType ********* matched on document 2021-10-01 ******************* Allgemeine Geschäftsbedingungen because it contains this string: "*********"

[2023-07-29 12:03:05,201] [DEBUG] [paperless.matching] DocumentType ********* matched on document 2021-10-01 ******************* Allgemeine Geschäftsbedingungen because it contains this word: *********

[2023-07-29 12:03:05,217] [DEBUG] [paperless.matching] DocumentType ********* matched on document 2021-10-01 ******************* Allgemeine Geschäftsbedingungen because it contains this word: *********

[2023-07-29 12:03:05,265] [INFO] [paperless.handlers] Detected 5 potential document types, so we've opted for *********

[2023-07-29 12:03:05,266] [INFO] [paperless.handlers] Assigning document type ********* to 2021-10-01 ******************* Allgemeine Geschäftsbedingungen

[2023-07-29 12:03:05,411] [DEBUG] [paperless.matching] Tag ****** matched on document 2021-10-01 ******************* Allgemeine Geschäftsbedingungen because it contains this word: ******

[2023-07-29 12:03:05,420] [INFO] [paperless.handlers] Tagging "2021-10-01 ******************* Allgemeine Geschäftsbedingungen" with "******"

[2023-07-29 12:03:05,787] [DEBUG] [paperless.consumer] Deleting file /tmp/paperless/paperless-ngxo_esd8o_/Allgemeine Geschäftsbedingungen.pdf

[2023-07-29 12:03:06,201] [DEBUG] [paperless.parsing.tesseract] Deleting directory /tmp/paperless/paperless-3dnd_f_7

[2023-07-29 12:03:06,202] [INFO] [paperless.consumer] Document 2021-10-01 ******************* Allgemeine Geschäftsbedingungen consumption finished

[2023-07-29 12:03:28,080] [DEBUG] [paperless.matching] Correspondent ******************* matched on document 2021-10-01 ******************* Allgemeine Geschäftsbedingungen because it contains this string: "*******************"

[2023-07-29 12:03:28,255] [DEBUG] [paperless.matching] Tag ****** matched on document 2021-10-01 ******************* Allgemeine Geschäftsbedingungen because it contains this word: ******

[2023-07-29 12:03:28,389] [DEBUG] [paperless.matching] DocumentType ********* matched on document 2021-10-01 ******************* Allgemeine Geschäftsbedingungen because it contains this string: "*********"

[2023-07-29 12:03:28,391] [DEBUG] [paperless.matching] DocumentType ********* matched on document 2021-10-01 ******************* Allgemeine Geschäftsbedingungen because it contains this string: "*********"

[2023-07-29 12:03:28,446] [DEBUG] [paperless.matching] DocumentType ********* matched on document 2021-10-01 ******************* Allgemeine Geschäftsbedingungen because it contains this string: "*********"

[2023-07-29 12:03:28,497] [DEBUG] [paperless.matching] DocumentType ********* matched on document 2021-10-01 ******************* Allgemeine Geschäftsbedingungen because it contains this word: *********

[2023-07-29 12:03:28,514] [DEBUG] [paperless.matching] DocumentType ********* matched on document 2021-10-01 ******************* Allgemeine Geschäftsbedingungen because it contains this word: *********

[2023-07-29 12:03:28,867] [DEBUG] [paperless.matching] Correspondent ******************* matched on document 2021-10-01 ******************* Allgemeine Geschäftsbedingungen because it contains this string: "*******************"

[2023-07-29 12:03:29,028] [DEBUG] [paperless.matching] Tag ****** matched on document 2021-10-01 ******************* Allgemeine Geschäftsbedingungen because it contains this word: ******

[2023-07-29 12:03:29,148] [DEBUG] [paperless.matching] DocumentType ********* matched on document 2021-10-01 ******************* Allgemeine Geschäftsbedingungen because it contains this string: "*********"

[2023-07-29 12:03:29,150] [DEBUG] [paperless.matching] DocumentType ********* matched on document 2021-10-01 ******************* Allgemeine Geschäftsbedingungen because it contains this string: "*********"

[2023-07-29 12:03:29,201] [DEBUG] [paperless.matching] DocumentType ********* matched on document 2021-10-01 ******************* Allgemeine Geschäftsbedingungen because it contains this string: "*********"

[2023-07-29 12:03:29,250] [DEBUG] [paperless.matching] DocumentType ********* matched on document 2021-10-01 ******************* Allgemeine Geschäftsbedingungen because it contains this word: *********

[2023-07-29 12:03:29,265] [DEBUG] [paperless.matching] DocumentType ********* matched on document 2021-10-01 ******************* Allgemeine Geschäftsbedingungen because it contains this word: *********

[2023-07-29 12:04:07,180] [DEBUG] [paperless.parsing.tesseract] Calling OCRmyPDF with args: {'input_file': PosixPath('/usr/src/paperless/media/documents/originals/*******************/2021/*******************_2021-10-01_Allgemeine Geschäftsbedingungen.pdf'), 'output_file': PosixPath('/tmp/paperless/paperless-o09r0r3y/archive.pdf'), 'use_threads': True, 'jobs': 2, 'language': 'deu', 'output_type': 'pdfa', 'progress_bar': False, 'skip_text': True, 'clean': True, 'deskew': True, 'rotate_pages': True, 'rotate_pages_threshold': 12.0, 'sidecar': PosixPath('/tmp/paperless/paperless-o09r0r3y/sidecar.txt')}

[2023-07-29 12:04:07,983] [DEBUG] [paperless.parsing.tesseract] Incomplete sidecar file: discarding.

[2023-07-29 12:04:08,304] [DEBUG] [paperless.parsing] Execute: convert -density 300 -scale 500x5000> -alpha remove -strip -auto-orient /tmp/paperless/paperless-o09r0r3y/archive.pdf[0] /tmp/paperless/paperless-o09r0r3y/convert.webp

[2023-07-29 12:04:10,604] [DEBUG] [paperless.parsing.tesseract] Deleting directory /tmp/paperless/paperless-o09r0r3y

[2023-07-29 12:04:17,058] [DEBUG] [paperless.matching] Correspondent ******************* matched on document 2021-10-01 ******************* Allgemeine Geschäftsbedingungen because it contains this string: "*******************"

[2023-07-29 12:04:17,227] [DEBUG] [paperless.matching] Tag ****** matched on document 2021-10-01 ******************* Allgemeine Geschäftsbedingungen because it contains this word: ******

[2023-07-29 12:04:17,347] [DEBUG] [paperless.matching] DocumentType ********* matched on document 2021-10-01 ******************* Allgemeine Geschäftsbedingungen because it contains this string: "*********"

[2023-07-29 12:04:17,349] [DEBUG] [paperless.matching] DocumentType ********* matched on document 2021-10-01 ******************* Allgemeine Geschäftsbedingungen because it contains this string: "*********"

[2023-07-29 12:04:17,405] [DEBUG] [paperless.matching] DocumentType ********* matched on document 2021-10-01 ******************* Allgemeine Geschäftsbedingungen because it contains this string: "*********"

[2023-07-29 12:04:17,456] [DEBUG] [paperless.matching] DocumentType ********* matched on document 2021-10-01 ******************* Allgemeine Geschäftsbedingungen because it contains this word: *********

[2023-07-29 12:04:17,471] [DEBUG] [paperless.matching] DocumentType ********* matched on document 2021-10-01 ******************* Allgemeine Geschäftsbedingungen because it contains this word: *********

[2023-07-29 12:05:00,893] [DEBUG] [paperless.classifier] Gathering data from database...

[2023-07-29 12:05:03,731] [DEBUG] [paperless.classifier] 746 documents, 17 tag(s), 1 correspondent(s), 4 document type(s). 0 storage path(es)

[2023-07-29 12:05:03,732] [DEBUG] [paperless.classifier] Vectorizing data...
```

### Paperless-ngx version

1.16.5

### Host OS

Synology DSM 7.2

### Installation method

Docker - official image

### Browser

Firefox 115.02 (64-bit)

### Configuration changes

PAPERLESS_OCR_LANGUAGE=deu PAPERLESS_CONSUMER_RECURSIVE=true PAPERLESS_CONSUMER_SUBDIRS_AS_TAGS=true

**Comment by shamoon** [comment]
This is https://github.com/paperless-ngx/paperless-ngx/issues/3743 and there’s not much we can do as the issue is the dependency we use for pdf display, there’s an upstream PR you can follow. It seems to be only (mainly) in FF and you can switch to the native viewer (that’s why the hover preview works)

**Comment by shamoon** [comment]
This should be fixed in 1.17.3

<img width="1112" alt="Screenshot 2023-08-28 at 9 37 36 PM" src="https://github.com/paperless-ngx/paperless-ngx/assets/4887959/16f0c44f-fa26-433c-a9bb-adeb28050fe9">
